lib/luhn_cases.rb

```diff
@@ -1,8 +1,6 @@
-require 'exercise_cases'
-
 class LuhnCase < ExerciseCase
   def workload
-    %Q(#{assert_or_refute} Luhn.valid?(#{input.inspect}))
+    "#{assert} Luhn.valid?(#{input.inspect})"
```

Comment:
It's a good idea to keep your commits atomic. 
This has nothing to do with underscores and should be in its own commit.

It is a discipline that needs to be trained. Commit early, commit often.